.changeset/beige-results-flow.md

```diff
@@ -0,0 +1,5 @@
+---
+"@cloudflare/vite-plugin": patch
+---
+
+fix: populate build time env var from `.env`
```

Comment:
```suggestion
fix: populate Hyperdrive local connection string from `.env`
```

Reply:
This should probably also include an example but I think we should land https://github.com/cloudflare/workers-sdk/pull/10652 first.